src/shellstrings.c

```diff
@@ -7,88 +7,331 @@
+               
+               "\nCurrently, you are in interactive mode. Return to the previous page to input a file.\n"
+               
+               "\nTo run in batch mode, exit the program with the command 'exit', "
+               "and run: '$ ./spellcheck [-flag] /path/file.txt'.\n"
+               
+               "\nFlags: '-q' is for quiet, and '-v' is for verbose.\n"
+               
+               "\nTo input a custom dictionary to the program, use " 
+               "'-d /path/dictionary.txt', and to provide a file saving destination, "
+               "use '-s /path/filename.txt'.\n"
+               
+               "\nTo enable color inside the program, use the '-c' flag.\n"
+
+               "\n-- Enter any command to return to the previous page. --\n");
+    }
 }
 
-/* See shellstrings.h */
-void shell_help() {
-    printf("Spellcheck is a tool which searches for misspelled words and suggests alternative spellings given a text file.\n");
-    printf("For interactive mode, run: r [/path/file.txt]\n");
-    printf("For batch mode, exit the program, and run: $ ./spellcheck [/path/file.txt] [-flag]\n");
-    printf("Flags:  -q is for quiet, -v is for verbose, and -s is for saving a file to a custom destination.\n");
-    printf("Enter any command to return to the previous page.\n");
+void shell_start_interactive(char *filename, char *dict, char *md, bool *color) {
+    if (*color == true) {   // only printing this out in verbose
+        log_info(BOLDWHITE "\n\n=============================================================\n"
+                               "====================== Editing Started ======================\n"
+                               "=============================================================\n\n" RESET);
+
+        log_info(BOLDWHITE "File         :" RESET " %s\n"
+                 BOLDWHITE "Dictionary   :" RESET " %s\n"
+                 BOLDWHITE "Mode         :" RESET " %s\n", filename, dict, md);
+    }
+
+    else {
+        log_info("\n\n=============================================================\n"
+                     "====================== Editing Started ======================\n"
+                     "=============================================================\n\n");
+
+        log_info("File         : %s\n"
+                 "Dictionary   : %s\n"
+                 "Mode         : %s\n", filename, dict, md);
+    }
 }
 
-/* See shellstrings.h */
-void shell_verbose_chart(int lnum, char* misspelled, char** suggestions) {
-    printf("%d\t\t\t", lnum); // print line number
+char *shell_modename(int mode) {
+    switch (mode) {
+        case 1:
+            return "Quiet Batch Mode";
+        case 2:
+            return "Verbose Batch Mode";
+        case 3: 
+            return "Interactive Mode";
+        default:
+            break;
+    }
+
+    return "Interactive Mode";
+}
+
+void shell_interactive_line_print(int lnum, char *line, char *underline, bool returnflag, bool *color) {
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!